gdk4/src/content_provider.rs

```diff
@@ -0,0 +1,29 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use crate::ContentProvider;
+use glib::translate::*;
+use glib::IsA;
+
+pub trait ContentProviderExtManual {
```

Comment:
you need to add the new trait to the prelude

Reply:
right, done